.rubocop.yml

```diff
@@ -1,9 +1,11 @@
 AllCops:
-   Exclude:
-     - spec/dummy/**/*
-     - script/*
-     - Guardfile
-     - vendor/**/*
+  Exclude:
+    - spec/dummy/**/*
+    - script/*
+    - Guardfile
+    - vendor/**/*
+    - lib/paperclip/*
+    - app/services/**/*
```

Comment:
I would prefer to leave this directory under the supervision of rubocop.

Reply:
done https://github.com/geminimvp/solidus_product_feed/pull/11/files#diff-11a0d7906801d9dea0eccb85667ad811R1